spec/features/two_factor_authentication/sign_in_spec.rb

```diff
@@ -278,8 +139,7 @@
     scenario 'user disables 2FA method' do
       user = sign_in_and_2fa_user
 
-      user.second_factors = []
-      user.save
+      user.update(mobile: nil, mobile_confirmed_at: nil)
```

Comment:
We can actually get rid of this entire scenario. It is not valid. A User can't remove their mobile once they've set it up.